lua/weapons/weapon_laserpointer.lua

```diff
@@ -0,0 +1,665 @@
+
```

Comment:
Please format https://goonlinetools.com/lua-beautifier/

you need to remove gmod-specific syntax (!, !=, &&, ||) and replace with standard syntax (not, ~=, and, or) for it to work